openupgrade_scripts/scripts/stock/16.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,98 @@
+from openupgradelib import openupgrade
+
+_models_renames = [
+    ("stock.location.route", "stock.route"),
+    ("stock.production.lot", "stock.lot"),
+]
+
+_tables_renames = [
```

Comment:
these should be added for the script to work, otherwise an error happens when the `stock` module is loaded because it creates a new `stock_route` table. `openupgrade.rename_models()` does not rename the tables.

```
    ("stock_location_route", "stock_route"),
    ("stock_production_lot", "stock_lot"),
```